docs/runtimes/dask-pipeline.ipynb

```diff
@@ -39,8 +39,7 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    def init_functions(functions: dict, project=None, secrets=None):
</pre><p>note to developers:</p><p><code>init_functions</code> are deprecated. we should fix the demo.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
there's a JIRAt: https://jira.iguazeng.com/browse/ML-3369